pkg/volume/vsphere_volume/attacher_test.go

```diff
@@ -308,7 +309,7 @@ func (testcase *testcase) DiskIsAttached(diskName string, nodeName types.NodeNam
 	return expected.isAttached, expected.ret
 }
 
-func (testcase *testcase) DisksAreAttached(diskNames []string, nodeName types.NodeName) (map[string]bool, error) {
+func (testcase *testcase) DisksAreAttached(nodeVolumes map[k8stypes.NodeName][]string) (map[k8stypes.NodeName]map[string]bool, error) {
```

Comment:
does it mean we cannot test this function for now?

Reply:
We are leaving this tests for now and planning to E2E test cases. I will have a few set of E2E test to verify this.